packages/components/side_nav/src/side_nav.tsx

```diff
@@ -0,0 +1,25 @@
+import { Grid } from "@buoysoftware/anchor-layout";
+import { SideNavLogo } from "./side_nav_logo";
+
+interface OwnProps {
+  children?: React.ReactNode;
+}
+
+type SideNavProps = OwnProps;
+
+export const SideNav: React.FC<SideNavProps> = ({
+  children,
+}): React.ReactElement => {
+  return (
+    <Grid
+      gridTemplateRows="auto 1fr auto"
+      p="xl"
+      bg="brandPrimary"
+      width="240px"
+      height="100%"
```

Comment:
I think you may have more consistent results with 100vh here
```suggestion
      height="100vh"
```

Reply:
Ooh nice!